index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+
+  <header class="container round">
+      <nav class="toolbar">
+        <div class="logo space blue">
+          My logo
+        </div>
```

Comment:
- [ ] Logo is looking good, great job! Let's make it even better using an `a` tag in order to make it clickable.